pkg/nsx/services/securitypolicy/parse.go

```diff
@@ -36,11 +36,3 @@ func getRuleDirection(rule *v1alpha1.SecurityPolicyRule) (string, error) {
 	}
 	return "", errors.New("invalid rule direction")
 }
-
-func getCluster(service *SecurityPolicyService) string {
```

Comment:
I suggest that we keep getCluster, the function is more helpful for future extension.  Also, we can consider put it as common function if necessary for all services.